src/if_ruby.c

```diff
@@ -982,6 +1021,21 @@ static void error_print(int state)
 	    if (p) *p = '\0';
 	    EMSG(buff);
 	}
+
+	attr = syn_name2attr((char_u *)"Error");
+# ifdef RUBY21_OR_LATER
+	bt = rb_funcallv(error, rb_intern("backtrace"), 0, 0);
+	for (i = 0; i < RARRAY_LEN(bt); i++)
+	{
```

Comment:
The convention is to not use the curly braces, if the "if" block contains
only a single statement.

Reply:
I removed the curly braces. Thank you! 90fadf2